index.bs

```diff
@@ -40,21 +40,17 @@ the default shipping configuration of user agents. They must only be
 enabled in testing configurations for example with special build
 flags, or when a specific non-default preference is set.
 
-The TestUtils Object {#the-testutils-object}
+The TestUtils namespace {#the-testutils-object}
```

Comment:
Change the link name, and use capital N for namespace.

Reply:
```suggestion
The TestUtils Namespace {#the-testutils-namespace}
```

Isn't the point of explicit link to keep it same regardless of the title change? Anyway, doing so...

Reply:
Yeah, but in practice I don't think there are many existing links, and a small amount of breakage now is probably better than having a misleading fragment id for all time.